### DISCUSSION TITLE
Where is url for the performance and cpu ?

### DISCUSSION THREAD

**Discussion by dranreb38** [jprochazka/adsb-receiver#603] [General]
I am testing your latest project, but I can't find the url to access the graph and performance
RPI3 and adsb-receiver]Release v2.8.6

**Comment by jprochazka** [comment]
I am guessing you found it?
Under System, Performance Graphs.
http://<RECEIVER_ADDRESS>/graphs.php

**Reply by dranreb38** [reply]
Thank you, I relaunched an install with less feeder because I encountered errors during the installation, and suddenly the script did not finish correctly!. Thank you for your response
